docker-compose.yaml

```diff
@@ -0,0 +1,23 @@
+version: '2'
+services:
+  zookeeper:
+    image: ghcr.io/theg-r-itters/zookeeper:latest
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
+    ports:
+      - 22181:2181
+  
+  kafka:
+    image: ghcr.io/theg-r-itters/kafka:latest
+    depends_on:
+      - zookeeper
+    ports:
+      - 29092:29092
+    environment:
```

Comment:
could be better use a env file

Reply:
Done

Reply:
You added the values in a file and use them in the docker compose env var. It's correct but you can add the env file directly with `env_file` tag in docker compose file, in this way it will be cleaner